v20.2 functions and operators refresh

ericharmeling opened the pull request:
Refreshing with latest functions/operators.

@rmloveland adding you to review, as most (if not all) of these new functions touch spatial features and will likely resolve issues currently assigned to you.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/docs/8543)
<!-- Reviewable:end -->

cockroach-teamcity commented:
Online preview: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/dd6485048bdbc9849bb6bea5fac88cc1e696879c/

rmloveland reviewed (approved):
Eric this LGTM - modulo that the broken build is because I added some direct anchor links like "operator-less-than" which are being pointed to from elsewhere. I think if you leave those anchors in everything else LGTM

ericharmeling commented:
> Eric this LGTM - modulo that the broken build is because I added some direct anchor links like "operator-less-than" which are being pointed to from elsewhere. I think if you leave those anchors in everything else LGTM

TFTR @rmloveland ! I went ahead and changed those references to match the generated functions page. I don't think the manual work of adding in those anchors is worth the accuracy of the links on the operator page.

cockroach-teamcity commented:
Online preview: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b676817302c9e1ecf963dfc75bf2d2af86568fae/

Edited pages:
- [v20.2/array.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b676817302c9e1ecf963dfc75bf2d2af86568fae/v20.2/array.html)
- [v20.2/install-cockroachdb-spatial.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b676817302c9e1ecf963dfc75bf2d2af86568fae/v20.2/install-cockroachdb-spatial.html)
- [v20.2/inverted-indexes.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b676817302c9e1ecf963dfc75bf2d2af86568fae/v20.2/inverted-indexes.html)
- [v20.2/spatial-data.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b676817302c9e1ecf963dfc75bf2d2af86568fae/v20.2/spatial-data.html)
- [v20.2/spatial-features.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b676817302c9e1ecf963dfc75bf2d2af86568fae/v20.2/spatial-features.html)
- [v20.2/spatial-glossary.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b676817302c9e1ecf963dfc75bf2d2af86568fae/v20.2/spatial-glossary.html)
- [v20.2/vectorized-execution.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b676817302c9e1ecf963dfc75bf2d2af86568fae/v20.2/vectorized-execution.html)
